storage/mounts.go

```diff
@@ -86,9 +92,9 @@ func (mp MountPoint) Mount(mo Mount, client *common.DatabricksClient) (source st
 				except Exception as e2:
 					print("Failed to unmount", e2)
 				raise e
-		mount_source = safe_mount("/mnt/%s", "%v", %s)
+		mount_source = safe_mount("/mnt/%s", "%v", %s, "%s")
 		dbutils.notebook.exit(mount_source)
-	`, mp.name, mo.Source(), extraConfigs)
+	`, mp.name, mo.Source(), extraConfigs, mp.encryptionType) // lgtm [go/unsafe-quoting]
```

Comment:
`lgtm [go/unsafe-quoting]` -> `lgtm[go/unsafe-quoting]`, as linter still complains

Reply:
linter still complains because it will be fixed only after merging - they are writing that in the message...